### PR TITLE
Update dependencies to allow newer versions of antlr4 and pytool

### PR DESCRIPTION
**Pull request by ghost**
There's no reason that edk2-basetools requires antlr4-python3-runtime version 4.7.1 or edk2-pytool-library version 0.10.13 specifically, so relax the requirements to allow newer versions of those packages.

**Note** I'm not entirely familiar yet with Python packaging or the edk2-basetools so I might be wrong.
Please close this PR if those versions really are required.

### CODE REVIEW

**Comment by bcran** [comment]
ping @lgao4 @mdkinney

**Comment by lgao4** [comment]
For antlr4-python3-runtime, I am OK for this change. 

For edk2-pytool-library , can Michael confirm it? @makubacki

File: requirements.txt [hunk 1]
```diff
@@ -3,4 +3,4 @@ pytest-html
 pytest-cov
 flake8
 antlr4-python3-runtime
-edk2-pytool-library==0.10.13
\ No newline at end of file
+edk2-pytool-library>=0.10.13
```
Comment (makubacki) [inline requirements.txt:6]: edk2-pytool-library does not use major versions and updates the minor version seldomly when there is a breaking change.

I recommend setting the version to the latest release [v0.21.9](https://github.com/tianocore/edk2-pytool-library/releases/tag/v0.21.9)

And use the `~=` operator to match patch versions, which typically run a while. For example, the current patch version is `9`. Then you can deal with any larger changes (minor or major) when they're brought to your attention through a mechanism like dependabot at which time you can evaluate if any integration changes are required and PR feedback of the incoming change.

```suggestion
edk2-pytool-library~=0.21.9
```

Comment (bcran) [reply requirements.txt]: Thanks!

**Comment by bcran** [comment]
I've made the suggested change to edk2-pytool-library. @makubacki  and @lgao4 could you approve the review please?

**Review by makubacki — APPROVED** [review]

**Pull request merged**